cea/inputlocator.py

```diff
@@ -49,6 +49,77 @@ def get_optimization_slave_results_folder(self):
         """
         return self._ensure_folder(self.get_optimization_results_folder(), "slave")
 
+    def get_optimization_slave_StorageOperationData(self, configkey):
```

Comment:
the names in the optimization code are non-PEP8 historically. I'd rather we refactored them instead of letting the mess "bleed" into the rest of the code. Can we use `get_optimization_slave_storage_operation_data` instead? likewise for the other variables.